torrt/notifiers/telegram.py

```diff
@@ -32,13 +33,19 @@ def test_configuration(self):
 
     def send_message(self, msg):
         url = '%s%s/sendMessage' % (self.url, self.token)
-        response = requests.post(url, data={'chat_id': self.chat_id, 'text': msg})
-        if response.ok:
-            json_data = response.json()
-            if json_data['ok']:
-                LOGGER.info('Telegram message was sent to user %s' % self.chat_id)
+        try:
+            response = requests.post(url, data={'chat_id': self.chat_id, 'text': msg})
+        except RequestException as e:
+            LOGGER.error('Failed to send Telegram message: %s' % e)
```

Comment:
There's no need to format (``%``) messages beforehand, just pass arguments.

Reply:
Wow! Thank you! All my life I used format for logging :(